web-services-metadata/2.1/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta Web Services Metadata Specification 2.1"
+date: 2019-07-23T14:28:20+0200
+summary: "First release for Jakarta EE 8"
```

Comment:
I just noticed this while doing a "final review" of these Java SE PRs...  These Specifications are not really part of Jakarta EE 8.  And, this particular version is not updated for Jakarta EE 9 yet.  We're just providing the required base for Jakarta EE 9 updates.  So, should we just change this summary to be :  "First release for Jakarta EE" (and drop the version number for now)?  If we agree with this change, then this would apply to all of these Java SE PRs...

Reply:
Yes, I agree.

Reply:
changed to _Initial release for Jakarta Web Services Metadata_